src/js/modules/TileGroupSet.js

```diff
@@ -3,47 +3,34 @@ import PropTypes from 'prop-types';
 
 import TileGroup from 'js/modules/TileGroup';
 
-
-const TileGroupSet = ({ groups, tileKey, groupTitleSubPath }) =>  {
-
-  // gets the number of topics that have active services
-  const countOfGroups = groups.edges
-    .map((g) => g.node.services.edges.length > 0)
-    .reduce((total, amount) => amount ? total + 1 : total, 0);
-  const shouldTilesStackVertical = countOfGroups > 1;
-
+const TileGroupSet = ({ tileGroups, tag }) =>  {
   return (
-    <div className="coa-TileGroupSet row">
-    {
-      groups && groups.edges.map(({node: group}, index) => {
-        if (group.services.edges.length < 1) return false;
-
-        const tiles = group[tileKey].edges.map((tile) => ({
-          url: `/${tileKey}/${tile.node.slug}`,
-          text: tile.node.title,
-        }));
-
-        return (
-          <TileGroup tiles={tiles}
-            tag="service"
-            hasBorder={true}
-            title={group.text}
-            key={index}
-            titlePath={`/${groupTitleSubPath}/${group.slug}`}
-            description={group.description}
-            direction={shouldTilesStackVertical ? 'column' : false}
-          />
-        )
-      })
-    }
+    <div className="coa-TileGroupSet">
+      <div className="row">
+      {
+        tileGroups.map(({ tiles, text, url, description }, index) => {
+          if (!tiles.length) return null;
```

Comment:
This is a n00b question. Why is returning `null` better than returning `false`?

Reply:
mmmmm, this is just a personal preference. I think of null as an empty type. Where as false is its own data type -- boolean.  so whenever my functions are expected to return a particular data type (in this case JSX), if it can't return that type, I like to return an empty vs a whole other data type.